src/tests/test_decorators.py

```diff
@@ -0,0 +1,39 @@
+from . import base
+from mock import patch
+import decorators
+
+class TestDecorators(base.BaseCase):
+    def test_timeit_smoke_test(self):
+        @decorators.timeit
+        def some_task():
+            pass
+        some_task()
```

Comment:
smoke test in the sense of "does not explode with runtime errors" :)
I think here a `@patch` can be used to capture the output and actually check it